lib/contestant.rb

```diff
@@ -0,0 +1,24 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age,
+              :state_of_residence, :spending_money, :game_interest
+    def initialize (info)
```

Comment:
this is throwing a syntax error-- you can't have spaces between a method and its parameters.